packages/components/src/internal/components/domainproperties/DomainForm.tsx

```diff
@@ -1140,37 +1155,39 @@ export class DomainFormImpl extends React.PureComponent<IDomainFormInput, IDomai
                             placeholder="Search Fields"
                             onChange={this.onSearch}
                         />
+
+                        {!testMode &&
+                            <ToggleWithInputField
+                                active={summaryViewMode}
+                                id={'domain-toggle-summary-' + domainIndex}
+                                onClick={this.onToggleSummaryView}
+                                on="Summary mode"
+                                off="Detail mode"
+                                style={{height: '34px', marginLeft: '10px'}}
```

Comment:
Sorry, should have called this out as well! This was kind of tricky.

The style here is passed to the <ReactBootstrapToggle/>, which expects styling to be passed in through this prop if it is to override the default styling on the toggle button (which sets a height of 20px). <ToggleWithInputFieldProps/>'s `containerClassName` prop does not apply to the button, but to the parent span. In <ReactBootstrapToggle/> there are props you can pass for classnames to smaller bits, like the 'on' button and the 'off' button, but no classname prop to pass to the element I need to style.
So, I end up having to use that style prop, but I agree it looks gross. I was wondering whether to pull out `{height: '34px', marginLeft: '10px'}` into a constant I then store in `/constants`, but wondered if that 'distance' would only end up being minorly inconvenient instead of a useful modularization. Do you have thoughts on this?

In the toolbar, the height (at least in the rightmost `col-xs-8` section) is not getting set directly but instead is dynamically internally controlled by the height of the search field, which is a `.form-control` and so a hard 27px. Which means we can't inherit or 100% a height into the toggle button, since the parents' height is dynamic instead of hard. I ended up getting around this using flexboxes to look good across platforms, but lmk of course if you see a more elegant solution!